modules/openy_gc_auth/modules/openy_gc_auth_daxko_barcode/src/Form/VirtualYDaxkoBarcodeLoginForm.php

```diff
@@ -49,10 +59,11 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->configFactory->get('openy_gc_auth.provider.daxco_barcode');
 
     $form['barcode'] = [
-      '#title' => $this->t('Barcode'),
-      '#description' => '',
+      '#title' => $this->t('@form_label', array('@form_label' => $config->get('form_label'))),
+      '#description' => $this->t('@form_description', array('@form_description' => $config->get('form_description'))),
```

Comment:
translation of @form_description makes no sense. Let's avoid using this here
Same for @form_label

Reply:
I have removed the translation and updated the code. Can you please verify that now

Reply:
thank you